src/Pos/Wallet/Redirect.hs

```diff
@@ -78,7 +78,7 @@ networkChainDifficultyWebWallet = getLastKnownHeader >>= \case
         cSlot <- flattenSlotId <$> MaybeT getCurrentSlot
         th <- lift (getTipHeader @ssc)
         let hSlot = flattenEpochOrSlot th
-        when (hSlot <= cSlot - blkSecurityParam) $
+        when (hSlot + fromIntegral blkSecurityParam <= cSlot) $
```

Comment:
The task was to prohibit summing number of blocks and number of slots, but that's exactly what is done here :(